app.js

```diff
@@ -2,17 +2,118 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
+// San Francisco [2 * 3] OI was a little suprised wse could evealute expressions inside of the brackets
+
+// !true && !(false || false) 
+// I'm assuming it's taking the ! and adding it to both the false booleans in the parathesis. Still suprising though. 
+
+//2.5 * 'fish'
+
+// I would have though JS would try to convert the number into a string like it does when adding and it did not. 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
 
-// List 1 expression you are still 'shaky' on, why?
+//  14 % 3 === 0
+//2 + 'pizza'
 
+// List 1 expression you are still 'shaky' on, why?
 
+//!true && !(false || false)
```

Comment:
You can pretty much ignore what's after the `&&` since the first thing is `!true` which means `false` Therefore the whole things will be false since `&&` is looking for both values to be true in order to return true